gulpfile.js/tasks/clean.js

```diff
@@ -3,7 +3,7 @@ var del    = require('del')
 var config = require('../config')
 
 var cleanTask = function (cb) {
-  del([config.root.dest]).then(function (paths) {
+  del([config.root.dest]).then(function () {
```

Comment:
I left it there as a reminder that the param is there to use if needed. I sometimes would log the paths to make sure I was targeting the right thing.

If you're removing it on your project, you could go a step further and shorten it to

```js
del([config.root.dest]).then(cb)
```